packages/react-native/ReactCommon/react/renderer/core/RawValue.h

```diff
@@ -183,6 +272,36 @@ class RawValue {
     return true;
   }
 
+  template <typename T>
+  static bool checkValueType(
+      const jsi::Value& value,
+      jsi::Runtime* runtime,
+      std::vector<T>* /*type*/) noexcept {
+    if (!value.isObject()) {
+      return false;
+    }
+
+    jsi::Object asObject = value.asObject(*runtime);
```

Comment:
If you used `isObject()` before, you can now use `getObject` instead of `asObject` here. `getObject` is faster because it uses an assert.